src/components/portal/common/SearchResults.js

```diff
@@ -12,7 +12,7 @@ class SearchResults extends Component {
     const { results } = this.props.product;
     if (!isEmpty(results)) {
       return results.filter(prod => prod.enabled !== false).map(prod => (
-        <div key={prod.productId} className="col-lg-3 imageGrid mt-3">
+        <div key={prod.productId} className="col-lg-2 imageGrid mt-3">
```

Comment:
pls not lg-2
Square images look bleh

Reply:
if not 2 its gonna be gigantic on big screen